plugins/autowarmth.koplugin/main.lua

```diff
@@ -160,18 +160,35 @@ end
 
 AutoWarmth._onEnterStandby = AutoWarmth._onSuspend
 
+function AutoWarmth:_onToggleNightMode()
+    if self.control_nightmode and not AutoWarmth.hide_nightmode_warning then
+        UIManager:show(ConfirmBox:new{
+            text = _("Night mode changed by the user.\nThe AutoWarmth plugin might change it again."),
```

Comment:
"by the user" strikes me as the code's/programmer's perspective ^_^
```suggestion
            text = _("Night mode changed.\nThe AutoWarmth plugin might change it again."),
```